src/spec.ts

```diff
@@ -55,7 +55,7 @@ export interface BaseSpec {
 }
 
 // TODO(https://github.com/vega/vega-lite/issues/2503): Make this generic so we can support some form of top-down sizing.
-export interface LayoutSize {
+export interface LayoutSizeMixins {
```

Comment:
Because I use plural elsewhere in the codebase.  Feel free to submit a PR to refactor all of them later.  (I agree that singular is more correct.)